src/module.ts

```diff
@@ -610,6 +619,21 @@ export default defineNuxtModule<ModuleOptions>({
     // @ts-ignore
     await nuxt.callHook('content:context', contentContext)
 
+    // Add sync components to use not global components
+    const components = (options.components || []).map(pascalCase)
+
+    addPluginTemplate({
```

Comment:
It would be better if we mark these components as global instead of registering a custom plugin. Marking them as global will help other modules to be list these components as global ones. Modules like `nuxt-component-meta`

Reply:
This feature is to avoid using global components and put them in the main bundle of the application actually. I want to see if this improves the performance

Reply:
I see, Nuxt registers lazy version of global components, but here the goal is to include it directly inside main bundle.

https://github.com/nuxt/nuxt/blob/ce9aecca4a39e1d5e36e6b92c77e706f8f577445/packages/nuxt/src/components/templates.ts#L50-L63